houseRobber.py

```diff
@@ -0,0 +1,50 @@
+class Solution(object):
+    #solution learned from discussion
+    def rob(self, nums):
+        """
+        : type nums: list[int]
+        : rtype: int
+        """
+        #f(-1) = 0 = preMax, f(0) = 0 = curMax
+        #use preMax to hold the max value of n - 2
+        #use curMax to hold the max value of n - 1
+        preMax = 0
```

Comment:
use snake_case instead of camelCase for local variables in python 
here an all over the file.